src/main/java/org/skriptlang/skript/bukkit/loottables/elements/expressions/ExprLootContextEntity.java

```diff
@@ -0,0 +1,91 @@
+package org.skriptlang.skript.bukkit.loottables.elements.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.entity.Entity;
+import org.bukkit.event.Event;
+import org.bukkit.loot.LootContext;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.loottables.LootContextCreateEvent;
+import org.skriptlang.skript.bukkit.loottables.LootContextWrapper;
+
+@Name("Loot Entity of Loot Context")
+@Description("Returns the loot context entity of a loot context.")
+@Examples({"set {_entity} to loot context entity of {_context}",
+	"",
+	"set {_context} to a new loot context at player:",
+		"\tset loot context luck value to 10",
+		"\tset loot context killer to player",
+		"\tset loot context entity to last spawned pig"
+})
+@Since("INSERT VERSION")
+public class ExprLootContextEntity extends PropertyExpression<LootContext, Entity> {
+
+	static {
+		Skript.registerExpression(ExprLootContextEntity.class, Entity.class, ExpressionType.PROPERTY,
+			"[the] loot [context] entity [of %-lootcontext%]",
```

Comment:
```suggestion
			"[the] loot [context] entity [of %lootcontext%]",
```
Should never be null

Reply:
Why? It’s null if they don’t use it and they are in the create event, so i allow something like `set loot context entity to blah` in the section

Reply:
this falls into why I prefered if you made the wrapper implement loot context, by making it not null but optional it will automatically default to the event reference

this is why things like `send ""` works in player events and commands